### ISSUE TITLE
freezes on boot XPS 13 2024

### ISSUE THREAD

**Issue by jorgenask86** [linuxmint/mint22.1-beta#93]
Hi! 
I have trouble booting after installing 22.1 BETA.
I have tried both 6.8 generic and oem kernels and even 6.11 oem

It works when I use recovery mode.

I am on a Dell XPS 13 9340 2024

I saw some mention of this in the blog but using nomodeset seems like a temporary solution

**Comment by clefebvre** [comment]
Can you boot Mint 22 and Ubuntu 24.04?

**Comment by jorgenask86** [comment]
Yes sorry I didnt mention that.. Linux Mint 22 and Ubuntu 24.04 boots fine.. even also Ubuntu 24.10

**Comment by clefebvre** [comment]
Try to downgrade the kernel to linux-image-6.8.0-38-generic (the version that shipped with 22).

**Comment by jorgenask86** [comment]
Tried to boot with linux-image-6.8.0-38-generic, but still only works with recovery mode.

So when I boot 22.1 from live usb it boots fine. and it gives me "built in display" under display settings.
As soon as I do the first reboot after installing 22.1 (and I have not installed any of the updates or done any changes) it will not boot unless with recovery mode or adding nomodeset to grub.

When I boot with recovery mode or with nomodeset, it says "unknown display" under display settings. So maybe it has to do with display driver.  My Dell XPS 13 has Intel Arc integrated graphichs I believe.

Many thanks for all the help and suggestions so far :)
Kind regards 
Jørgen

**Comment by jorgenask86** [comment]
Not sure if it helps but ubuntu cinnamon 24.10 also boots fine

**Comment by jorgenask86** [comment]
For what its worth, I tried the 22.10 LM Mate Beta and the problem is the same there

**Comment by clefebvre** [comment]
Could it be secureboot? Could it be the media codecs?

I don't have enough information to identify a bug here or to do anything.

**Comment by jorgenask86** [comment]
Secure boot is disabled and I tried now to install without the media codecs and same result, gets stuck with black screen on boot.
So not sure either whats going on
